askama_actix/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 
 [dependencies]
 actix-web = { version = "4", default-features = false }
-askama = { version = "0.11.2", path = "../askama", default-features = false, features = ["with-actix-web"] }
+askama = { version = "0.11", path = "../askama", default-features = false, features = ["with-actix-web"] }
```

Comment:
Why did you change this?

Reply:
because they should all be matching the latest update as those are for bug fixes rather than API additions or changes. unless they are not following any standard for incrementing the version? otherwise when he releases an update to askama that fixes a bug that did not change the api then he does not need to republish all of the smaller libraries pointing to askama and they all see the latest updates for regular bug fixes.

Reply:
@genusistimelord 0.11.2 is a minimum bound, the ^ semver operator is implicit in Cargo.toml. So this already supports anything 0.11.x with x >= 2.